app/invocation/invocation_overview.tsx

```diff
@@ -47,20 +47,34 @@ export default class InvocationOverviewComponent extends React.Component {
     router.navigateToSetup();
   }
 
+  userHasAccessToOrg() {
+    const invocation = this.props.model.invocations[0];
+    if (!invocation) return false;
+
+    return Boolean(this.props.user?.groups?.some((group) => group.id === invocation.acl.groupId));
+  }
+
+  getOwningGroupName() {
```

Comment:
looks like we gave this logic in 3 spots now, can we put this method in invocation_model.tsx and use thruthiness to tell whether or not user has access to org (this will return null/undefined if the user doesn't have access)

Reply:
Done - much cleaner now :+1: